test/options/lines-between-rulesets-css/test.js

```diff
@@ -0,0 +1,24 @@
+/* global describe, it */
+'use strict';
+
+describe('options/lines-between-rulesets:css', function() {
```

Comment:
@ahumphreys87 Is this the test you're looking for? Looks like it handles three rulesets on the same line.